### ISSUE TITLE
Open proxy when in disabled mode?

### ISSUE THREAD

**Issue by speegs** [engintron/engintron#197]
Hello,

I had engintron in disabled mode for 2 days.  I then discovered thousands of connections after investigating why I was getting data usage warnings from the VPS provider.

Seems engitron at least on my cpanel server (pretty new installed) decided to become an open proxy server running on port 8080 when disabled.

I imagine to fix it, simple telling the nginx service to stop after it swaps ports with apache might be a good idea.  Unless there is something more complex at play here.  Would love to know if others have seen the behaviour too or if a developer of engintron can verify it.

Thanks.

**Comment by speegs** [comment]
Decided to add a firewall rule blocking acccess to port 8080, but allow localhost.  Seems to be better now.  Not sure if that is the recommended config, will report back if I find that causes problems.

**Comment by fevangelou** [comment]
Since others asked, do you mind sharing your iptables rule?

**Comment by speegs** [comment]
I'm using a WHM Plugin which I guess generates the iptables rules.  So these are not in iptables format.

ConfigServer Security & Firewall

In the allow rules:
tcp|in|d=8080|s=ipofserver
tcp|in|d=8080|s=127.0.0.1

In the deny rules:
tcp|in|d=8080|s=0/0

I don't think it's required in more recent versions of engintron, not really sure it's been a long while since I've turned engintron off, I usually just modify the config to not cache certain url patterns now vs turning engintron off completely if I think I have a problem.   I got a little smarter.
